doc/tutorials/using_partition_mode_on_nuc.rst

```diff
@@ -20,6 +20,10 @@ Validated Versions
 - Ubuntu version: **18.04**
 - ACRN hypervisor tag: **v2.6**
 
+.. note:: After the v2.6 release, the ``logical_partition`` scenario
+   is renamed to ``partioned``, affecting the file names used in this
```

Comment:
partioned --> partitioned

Also, maybe "was renamed" instead of "is renamed"? Another maybe: "...this guide that was validated with v2.6" --> "...this guide (which was validated with v2.6)".